server/app/repository/UserRepository.java

```diff
@@ -189,6 +190,11 @@ public void addTrustedIntermediaryToGroup(long id, String emailAddress)
               a.save();
               return a;
             });
+
+    if (account.getGlobalAdmin() || !account.getAdministeredProgramNames().isEmpty()) {
```

Comment:
**question (non-blocking):** Is there a technical reason for preventing this?

Reply:
That's helpful context, thanks!

Reply:
Actually there is the UX reason of "which portal do we redirect an admin+TI user upon login" that we would have to resolve here if we allowed it - forgot about that.

Reply:
Aha, thanks for mentioning